front/app/containers/Authentication/Modal.tsx

```diff
@@ -60,8 +60,8 @@ const AuthModal = ({ setModalOpen }: ModalProps) => {
   const theme = useTheme();
 
   useEffect(() => {
-    setModalOpen?.(currentStep !== 'closed');
-  }, [currentStep, setModalOpen]);
+    onToggleModal?.(currentStep !== 'closed');
```

Comment:
Then we can adjust this to
```suggestion
    onToggleModal(currentStep !== 'closed');
```